packages/controls/src/widget_description.ts

```diff
@@ -69,19 +67,17 @@ class DescriptionView extends DOMWidgetView {
 
     updateDescription() {
         let description = this.model.get('description');
-        let description_tooltip = this.model.get('description_tooltip');
-        if (description_tooltip === null) {
-            description_tooltip = description;
-        }
-
         if (description.length === 0) {
             this.label.style.display = 'none';
         } else {
             this.label.innerHTML = description;
             this.typeset(this.label);
             this.label.style.display = '';
         }
-        this.label.title = description_tooltip;
+    }
+
+    updateTooltip() {
+        this.label.title = this.model.get('tooltip');
```

Comment:
Should this still fall back to using the `description` value if `tooltip` is unset?